### ISSUE TITLE
Verify that all UEFI plans can be used

### ISSUE THREAD

**Issue by displague** [equinix-labs/terraform-equinix-metal-eks-anywhere#33]
Use the plans.
Ensure they provision correctly and are usable.
Add the plans to the list of known good.

**Comment by cprivitere** [comment]
Couldn't find enough a3.larges to test.

**Comment by cprivitere** [comment]
n3.xlarges are failing even during a manual run of eksctl anywhere.

**Comment by cprivitere** [comment]
m3.larges fail on netdog during the boot of bottlerocket. Likely means we need a different network interface name in the m3.large tinkerbelltemplateconfig file.

**Comment by cprivitere** [comment]
Ok, status update: I have been unable to get m3.large, n3.xlarge, a3.large, or c2.medium to work. We need to work with some bottlerocket experts at AWS to get help debugging what's going wrong.

**Comment by displague** [comment]
We're going to try the following configuration change in an attempt to reduce the configuration problems that go along with variability in NIC kernel device naming:

```hcl
kernel {
  <console stuff goes here>
}
init {
  net.ifnames=0
}
```

**Comment by displague** [comment]
@Paulius0112 yes. The device name that is configured in this project and passed to Bottlerocket must match the kernel device name of the device. We opened #63 in an attempt to workaround this.

https://github.com/equinix-labs/terraform-equinix-metal-eks-anywhere/blob/main/variables.tf#L92-L99 declares a variable that you can use to send plan-specific kernel NIC device name in, you'll want to verify the name and add it to the map. 

m3.small.x86 is the most tested plan with this integration. There may be other limitations (gotchas) outside of this plan. 

We plan to revisit these issues after the next bottlerocket and EKS-A releases since bottlerocket's next release addresses the NIC naming concerns by allowing for MAC address mapping in addition to NIC name mapping.

**Comment by displague** [comment]
@Paulius0112 We didn't bake additional plans into the map because the NIC device name was variable beyond just knowing the plan name.  NIC device names are sensitive to the placement within the server (vendor, bus, slot, etc).

**Comment by displague** [comment]
We found this was very consistent with m3.small.x86.
